crates/uv/src/commands/project/run.rs

```diff
@@ -187,11 +187,42 @@ pub(crate) async fn run(
     // Discover and sync the base environment.
     let temp_dir;
     let base_interpreter = if let Some(script_interpreter) = script_interpreter {
+        // If we found a PEP 723 script and the user provided a project-only setting, warn.
+        if !extras.is_empty() {
+            warn_user_once!("Extras are not supported for Python scripts");
```

Comment:
I will change this to show specific CLI arguments when I do https://github.com/astral-sh/uv/issues/5855 (I'll need access to the args to fix that).